01_login.php

```diff
@@ -73,29 +95,46 @@
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
 			<div class="w3-bar w3-white w3-padding w3-card">
+				<div class="w3-left">
```

Comment:
placed logo on the left; did a test and preferred the placing better, looks more professional